src/components/Card/Card.tsx

```
@@ -0,0 +1,115 @@
+  const [isExpanded, setIsExpanded] = useState<boolean>(
+    initialIsExpanded ?? true,
+  );
+
+  const hasButton = !!buttonLabel && !!buttonOnClick;
+  const shouldShowChildren =
+    (isCollapsible && isExpanded) ?? externalIsExpanded ?? !isCollapsible;
+
+  return (
+    <VStack className="gap-4 rounded-lg border p-4 md:gap-4 lg:p-6">
+      <VStack as="header" className="gap-4">
+        {!!header ? (
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)